client/src/pages/locations/Form.js

```diff
@@ -84,6 +84,21 @@ const LocationForm = ({ edit, title, initialValues }) => {
     }
   }
 
+  // Location types to be shown to admins in the new location page.
+  const locationTypesAdmin = [
+    Location.LOCATION_TYPES.ADVISOR_LOCATION,
+    Location.LOCATION_TYPES.PRINCIPAL_LOCATION,
+    Location.LOCATION_TYPES.PINPOINT_LOCATION,
```

Comment:
Admins can't choose `PINPOINT_LOCATION` (it's only a parent type for advisor or principal location).

Reply:
AFAIR, it might be a valid case for NMI.

Reply:
We've decided to allow this for now. At some point, we may introduce a new sub-type of pinpoint location (e.g. PPU or PPO), but not right now.